SyncApp/Controllers/HomeController.cs

```diff
@@ -257,6 +261,43 @@ public async Task<ActionResult> ExportSalesAsync(bool fromWeb, DateTime dateToRe
             return View("~/Views/Home/ExportDailySales.cshtml", "N/A");
         }
 
+        private async void GetCountriesFromShopify()
```

Comment:
Because we are doing some DB changes here, you can add it in CountriesLogic and name for example: UpsertCountries or something like that (add or update)

Reply:
Done